src/components/molecules/OakModal/OakModal.tsx

```diff
@@ -58,6 +59,7 @@ const FadeOutBox = styled(OakBox)<{ $state: TransitionStatus }>`
 `;
 
 const SlideInFlex = styled(OakFlex)<{ $state: TransitionStatus }>`
+  max-width: calc(100vw - ${parseSpacing("inner-padding-l")});
```

Comment:
This ensures that the modal leaves a little space peeking through to the right when it slides out (matching the designs as much as is practical)